react/components/InstallmentsRenderer.tsx

```diff
@@ -3,9 +3,9 @@ import { FormattedNumber } from 'react-intl'
 import { useCssHandles } from 'vtex.css-handles'
 import { FormattedCurrency } from 'vtex.format-currency'
 import { IOMessageWithMarkers } from 'vtex.native-types'
+import { ProductTypes } from 'vtex.product-context'
```

Comment:
I didn't see the PR with this I think, but I don't like `ProductTypes`, why not just `Product`?

Reply:
I thought `ProductTypes` would be more intuitive. Kinda like the `types` import from `@babel/core` (https://babeljs.io/docs/en/babel-types). This were you can find type definitions for Babel's AST